lolautruche/payline-bundle/2.0/post-install.txt

```diff
@@ -0,0 +1,9 @@
+<bg=blue;fg=white>                                                 </>
+<bg=blue;fg=white> You just installed LolautruchePaylineBundle!    </>
+<bg=blue;fg=white> What's next?                                    </>
+<bg=blue;fg=white>                                                 </>
+
+  * <fg=blue>Configure</> Payline environment variables and settings in <comment>config/packages/payline.yaml</comment>.
```

Comment:
but the yaml file references env vars - so we should tell ppl to have a look at `.env` files instead isn't it?